src/xlsx/style_parser.rs

```diff
@@ -46,15 +57,28 @@ fn parse_color(attributes: &[Attribute]) -> Result<Option<Color>, XlsxError> {
 /// Parse font weight from string
 fn parse_font_weight(s: &str) -> FontWeight {
     match s {
-        "bold" => FontWeight::Bold,
-        _ => FontWeight::Normal,
+        "bold" | "700" => FontWeight::Bold,
```

Comment:
I think technically in the Xls format bold can have other weights but I haven't seen it in practice.